pkg/agent/openflow/network_policy.go

```diff
@@ -1979,17 +1994,17 @@ func (c *client) NetworkPolicyMetrics() map[uint32]*types.RuleMetric {
 	}
 	if c.enableMulticast {
 		// We need to collect NP statistics matching IGMP query messages and egress multicast traffic.
-		collectMetricsFromFlows(MulticastIngressMetricTable, parseMulticastMetricFlow)
-		collectMetricsFromFlows(MulticastEgressMetricTable, parseMulticastMetricFlow)
+		collectMetricsFromFlows(MulticastIngressMetricTable, nil, parseMulticastMetricFlow)
```

Comment:
Why it is nil for the verifyflow in MulticastIngressMetricTable/MulticastEgressMetricTable? They should be the same since all metric tables have similar logics.

I doubt your concern is that flows in multicast metric table has no condition with key "ct_label". If so, you could think about using a string slice as parameter for the key words?

Reply:
Yes, I will add another verifier for multicast to avoid using nil.